Library/Formula/gettext.rb

```diff
@@ -23,32 +24,36 @@ def install
     ENV.libxml2
     ENV.universal_binary if build.universal?
 
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--disable-debug",
-                          "--prefix=#{prefix}",
-                          "--with-included-gettext",
-                          "--with-included-glib",
-                          "--with-included-libcroco",
-                          "--with-included-libunistring",
-                          "--with-emacs",
-                          "--with-lispdir=#{share}/emacs/site-lisp/gettext",
-                          "--disable-java",
-                          "--disable-csharp",
-                          # Don't use VCS systems to create these archives
-                          "--without-git",
-                          "--without-cvs",
-                          "--without-xz"
-    system "make"
-    system "make", "check"
+    args = %W[
```

Comment:
Mind leaving the args and `make` sections alone in this PR?

Reply:
I shall do so at my earliest convenience.  Sorry for the mix-up about the purpose of the "make" convenience function.

Reply:
Edits made.  Is it possible to accept the revised PR against HEAD, or do I need to figure out some hocus-pocus to delete the sub-par first attempt that's currently in between them on this branch?